bundles/org.openhab.binding.mqtt.espmilighthub/src/main/java/org/openhab/binding/mqtt/espmilighthub/internal/handler/EspMilightHubHandler.java

```diff
@@ -0,0 +1,385 @@
+            }
+            return;
+        }
+    }
+
+    @Override
+    public void handleCommand(ChannelUID channelUID, Command command) {
+        if (command instanceof RefreshType) {
+            return;
+        }
+        switch (channelUID.getId()) {
+            case CHANNEL_LEVEL:
+                handleLevelColour(command);
+                return;
+            case CHANNEL_BULB_MODE:
+                bulbMode = command.toString();
+                break;
+            case CHANNEL_COLOURTEMP:
+                int scaledCommand = (int) Math.round((370 - (2.17 * Float.valueOf(command.toString()))));
+                sendMQTT("{\"state\":\"ON\",\"level\":" + savedLevel + ",\"color_temp\":" + scaledCommand + "}");
+                break;
+            case CHANNEL_COMMAND:
+                sendMQTT("{\"command\":\"" + command + "\"}");
+                break;
+            case CHANNEL_DISCO_MODE:
+                sendMQTT("{\"mode\":\"" + command + "\"}");
+                break;
+            case CHANNEL_COLOUR:
+                handleLevelColour(command);
+        }
+    }
+
+    @Override
+    public void initialize() {
+        config = getConfigAs(ConfigOptions.class);
+        if (config.dimmedCT > 0) {
+            maxColourTemp = new BigDecimal(config.favouriteWhite);
+            minColourTemp = new BigDecimal(config.dimmedCT);
+            if (minColourTemp.intValue() <= maxColourTemp.intValue()) {
+                updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_PENDING,
+                        "The dimmedCT config value must be greater than the favouriteWhite value.");
+            }
+        }
+        if (getBridge() == null) {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_PENDING,
+                    "Globe must have a valid bridge selected before it can come online.");
+        } else {
+            globeType = thing.getThingTypeUID().getId();// eg rgb_cct
+            String globeLocation = this.getThing().getUID().getId();// eg 0x014
+            remotesGroupID = globeLocation.substring(globeLocation.length() - 1, globeLocation.length());// eg 4
+            String remotesIDCode = globeLocation.substring(0, globeLocation.length() - 1);// eg 0x01
+            fullCommandTopic = COMMANDS_BASE_TOPIC + remotesIDCode + "/" + globeType + "/" + remotesGroupID;
+            fullStatesTopic = STATES_BASE_TOPIC + remotesIDCode + "/" + globeType + "/" + remotesGroupID;
+            // Need to remove the lowercase x from 0x12AB in case it contains all numbers
+            String caseCheck = globeLocation.substring(2, globeLocation.length() - 1);
+            if (!caseCheck.equals(caseCheck.toUpperCase())) {
+                logger.warn(
+                        "The milight globe {}{} is using lowercase for the remote code when the hub needs UPPERCASE",
+                        remotesIDCode, remotesGroupID);
+            }
+            Bridge localBridge = getBridge();
```

Comment:
Please move this variable to before line 290 so that you can use it there too.

Reply:
DONE, thanks for spotting.